python-frontend/src/main/java/org/sonar/python/semantic/SymbolTableBuilder.java

```diff
@@ -753,6 +754,26 @@ public void visitFunctionDef(FunctionDef functionDef) {
       super.visitFunctionDef(functionDef);
     }
 
+    @Override
+    public void visitAnnotatedAssignment(AnnotatedAssignment annotatedAssignment) {
```

Comment:
I am not sure I understand why we would need to do this visit in the ThirdPhase and not the Second one already? I am not sure I understand the purpose of the ThirdPhase could we have a quick chat about it?